Sources/TuistCore/GraphTraverser/GraphTraversing.swift

```diff
@@ -86,6 +86,8 @@ public protocol GraphTraversing {
     ///   - name: Target name.
     func directLocalTargetDependencies(path: AbsolutePath, name: String) -> Set<GraphTarget>
 
+    func combinedCondition(to transitiveDependency: GraphDependency, from rootDependency: GraphDependency) -> TargetDependency.Condition.CombinationResult
```

Comment:
This still needs comment docs.  maybe this shouldnt even be public but this was the MVP to test the fix.